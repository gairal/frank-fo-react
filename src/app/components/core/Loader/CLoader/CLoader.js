import React from 'react';
import { CircularProgress } from '@material-ui/core';

export default () => (
  <div style={{ textAlign: 'center', marginTop: '30vh' }}>
    <CircularProgress size={200} />
  </div>
);
