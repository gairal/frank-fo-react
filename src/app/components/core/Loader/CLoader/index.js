import CLoader from './CLoader';

export default CLoader;
