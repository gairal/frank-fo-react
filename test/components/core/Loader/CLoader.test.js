import React from 'react';
import renderer from 'react-test-renderer';
import CLoader from '@/components/core/Loader/CLoader';

describe('components > core > CLoader', () => {
  it('Renders correctly', () => {
    const component = renderer.create(<CLoader />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  // it('Appears when request fetching', () => {
  //   let component = renderer.create(<Loader />);
  //   let tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();

  //   component = renderer.create(<Loader isFetching />);
  //   tree = component.toJSON();
  //   expect(tree).toMatchSnapshot();
  // });
});
