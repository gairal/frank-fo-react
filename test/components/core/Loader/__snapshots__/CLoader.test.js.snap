// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components > core > CLoader Renders correctly 1`] = `
<div
  style={
    Object {
      "marginTop": "30vh",
      "textAlign": "center",
    }
  }
>
  <div
    className="MuiCircularProgress-root-1 MuiCircularProgress-colorPrimary-4 MuiCircularProgress-indeterminate-3"
    role="progressbar"
    style={
      Object {
        "height": 200,
        "width": 200,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg-6"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle-7 MuiCircularProgress-circleIndeterminate-9"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>
</div>
`;
